### PR TITLE
Fix images on the documentation

### PR DESCRIPTION
**Pull request by arkerone**
This PR should fixes the issue reported in #225

The SVG images are optimized and I get this 
image

Should I turn off optimization for the resource folder? or is this another issue?

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Review by jsumners — CHANGES_REQUESTED** [review]
> Should I turn off optimization for the resource folder?

Please turn off optimization. The SVG should not look like your screenshot. You can see what it should look like by viewing the same document in the main Fastify repo.

**Comment by arkerone** [comment]
> > Should I turn off optimization for the resource folder?
> 
> Please turn off optimization. The SVG should not look like your screenshot. You can see what it should look like by viewing the same document in the main Fastify repo.

I've fixed it!

**Review by jsumners — APPROVED** [review]
Amazing work! Thank you for fixing this big issue.

@lmammino does the local preview look right to you?

File: src/scripts/buildWebsite.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ Metalsmith(source)
   )
   .use(
     contenthash({
-      pattern: ['**/*.{js,css,png,jpg,svg}']
+      pattern: ['**/*.{js,css,png,jpg,svg}', '!**/resources/*.*']
```
Comment (lmammino) [inline src/scripts/buildWebsite.js]: Why are we disabling content hashing for these resources? If they change these assets, the old version will stay in the CDN cache and we won't be able to see the updated version.

**Review by lmammino — COMMENTED** [review]
I can confirm the SVG looks good locally. Thanks a lot, @arkerone for all the good work!

I am still concerned about the lack of content hashing though.

I'd say, either we find a way to reference the image directly from GitHub or, if we prefer to copy the asset into the website, then we need to keep content hashing to allow us to invalidate the cache in case we decide to update the image.

File: src/scripts/buildWebsite.js (hunk 1 above)
Comment (arkerone) [reply src/scripts/buildWebsite.js]: I didn't find out how to use the hash via the `hashes` metadata object. I'll investigate to resolve this problem.

**Comment by arkerone** [comment]
I've fixed it. We can use the hash version of images.

**Comment by arkerone** [comment]
I think everything seems good. Tell me if anything is missing or is a problem

**Comment by jsumners** [comment]
I'm sure @lmammino will re-review when he is able.

File: src/plugins/metalsmith-nunjucks-renderer.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const extname = require('path').extname
```
Comment (lmammino) [inline src/plugins/metalsmith-nunjucks-renderer.js:1]: Why do we need this plugin?

**Review by lmammino — APPROVED** [review]
Looks good! Thanks for the great contribution @arkerone!

I have left only one minor question

File: src/plugins/metalsmith-nunjucks-renderer.js (hunk 2 above)
Comment (arkerone) [reply src/plugins/metalsmith-nunjucks-renderer.js:1]: It's to use the hashed version of images via the `hashes` metadata object (ie: in html `{{hashes[url}}`). When the markdown is converted in html, I use a custom renderer for image and I transform the URL like that :`${title}`, then we need to use nunjucks to render the correct url via this hashes object which is injected in this plugin.

Comment (lmammino) [reply src/plugins/metalsmith-nunjucks-renderer.js:1]: Oh right! I think I see now. Very clever :)

**Comment by lmammino** [comment]
It's live! https://www.fastify.io/docs/latest/Encapsulation/

Thanks again @arkerone!

**Comment by arkerone** [comment]
> It's live! https://www.fastify.io/docs/latest/Encapsulation/
> 
> Thanks again @arkerone!

Great, good news  ! Thanks :+1:

**Pull request merged**